docs/fast_data/inputs_and_outputs.md

```diff
@@ -398,6 +402,10 @@ channels:
                 primaryKeys:
                   description: Array of the primary key field names
                   type: array
+                before:
+                  type: object
+                  description: Value of the MongoDB record **before** the changes have been applied. In case of a insert operation this field is not defined
+                  additionalProperties: true
```

Comment:
If we want to add it we need to specify in which cases is sent, like having db2 adapter or custom without giving the before, this will not be defined.